include/boost/date_time/gregorian/greg_year.hpp

```diff
@@ -30,7 +30,7 @@ namespace gregorian {
   //! Generated representation for gregorian year
   typedef CV::constrained_value<greg_year_policies> greg_year_rep;
 
-  //! Represent a day of the month (range 1900 - 10000) 
+  //! Represent a year (range 1400 - 10000) 
   /*! This small class allows for simple conversion an integer value into
       a year for the gregorian calendar.  This currently only allows a
       range of 1900 to 10000.  Both ends of the range are a bit arbitrary
```

Comment:
It's 1400, I missed the other place in the comment. You can see the valid range in the `greg_year_policies` definition above. I've updated the commit.